src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -399,9 +398,13 @@ class CompactBlockProcessor(
         }
     }
 
+    internal suspend fun refreshUtxos(tAddress: String, startHeight: Int): Int = withContext(IO) {
```

Comment:
This API doesn't work well with multiple account support.

Reply:
Very good point. Multiple account support is broken in a few places so I've spun this out into a separate github issue so that we don't lose the feedback:
https://github.com/zcash/zcash-android-wallet-sdk/issues/221